src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -298,6 +375,9 @@ private void EndListen(Exception unhandledException)
             }
         }
 
+        /// <summary>
+        /// Disposes resources used by the object.
+        /// </summary>
```

Comment:
Should we not name this Dispose? (and use the IDIsposable pattern?)
Well again I guess this is coming so later

Reply:
yes i'm absolutely aware of this little guy